firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -138,7 +138,7 @@ public InstallationResponse createFirebaseInstallation(
       @NonNull String projectID,
       @NonNull String appId,
       @Nullable String iidToken)
-      throws IOException {
+      throws FirebaseInstallationsException, IOException {
```

Comment:
Why are we still throwing `IOException`?
Can we get around this and only throw 1 exception type?
See comment below.